documentation/architecture/decisions/0004-cloudwatch-metrics.md

```diff
@@ -0,0 +1,83 @@
+ApplicationELB/RequestCount
+ApplicationELB/TargetResponseTime
+ApplicationELB/HTTPCode_ELB_5XX_Count
+EBS/VolumeWriteBytes
+EBS/VolumeReadBytes
+EBS/VolumeReadOps
+EBS/VolumeWriteOps
+```
+
+we appear to be requesting roughly 4000 metrics per scrape. If we scraped our current config at a period of once every 10 minutes we would end up roughly $21,000 a year. If we scraped it every 30 seconds it would
+be about $420,000.
+
+By requesting only ALB metrics in the dev accounts, we still produce about 160 API requests according to the `cloudwatch_requests_total` counter for each scrape. Somewhat strangely, this only returns about 30 timeseries so we are not sure if our config is incorrect and if the number of API calls could be reduced to closer
+match the number of timeseries.
+
+Note, as dev accounts have lots of resources e.g. volumes, there may be fewer
+metrics requested for staging and prod as unlike the dev account we wouldn't be
+exporting metrics for other stacks.
+
+It takes a long time to get a response from the /metrics endpoint as it needs to make many API calls. This causes slow response times for which our prometheus scrape config needs to allow for using the scrape_timeout setting
+
+For speed of development, we still are only running one instance per ECS cluster. There were questions raised about having such as CPU intensive process (that was prone to crashing due to CPU spikes) running on the same instance as prometheus. However we didn't do anything about this as it would require a lot of work to have more than one instance per cluster.
+
+Config race condition was encountered every time with config grabbing for the exporter meaning either a 2nd restart of the task needed or we would need to add a delay to the exporter starting. We did not try and solve this
+
+Exporter task is slow to start up, health check needs longer than usual to pass health checks
+
+We spotted that several targets we are scraping such as prom, alertmanager are set at very short scrape intervals (every 5s), this seems excessive and we can likely change down to every 30secs regardless of this story
+
+We found the Cloudwatch exporter app to be very CPU intensive, we had to double our instance size to a `m4.2xlarge` to get it to work. We were also concerned about having such a CPU intensive task running in the same instance as Prometheus.
+
+There is roughly a 15 minute delay in Cloudwatch metrics due to `CloudWatch has been observed to sometimes take minutes for reported values to converge. The default delay_seconds will result in data that is at least 10 minutes old being requested to mitigate this.` - from [Prometheus CloudWatch exporter README](https://github.com/prometheus/cloudwatch_exporter/blob/master/README.md).
+
+## Decision
+
+We've proven that we are able to get the metrics from Cloudwatch but due to the high cost of running the CloudWatch exporter the team decided not to continue working on this spike as it was not a viable long term solution. 
```

Comment:
can you write this as a clear decision, without any surrounding context.  The rest of the document is for context, but this section should just be what specifically we have decided. something like:

> We will not use the cloudwatch_exporter to gather Cloudwatch metrics into prometheus.